ext/POSIX/lib/POSIX.pm

```diff
@@ -247,6 +246,18 @@ sub sprintf {
     CORE::sprintf(shift,@_);
 }
 
+my @_atexit;
+
+END {
+    my $f;
+    $f->() while $f = pop @_atexit;
```

Comment:
I don't like that this loop stops when a false value is encountered in `@_atexit`.

Why not
```perl
for my $f (reverse @_atexit) {
    $f->();
}
```
or
```perl
pop(@_atexit)->() while @_atexit;
```
?

Reply:
> I don't like that this loop stops when a false value is encountered in @_atexit.
I don't intent put false values into `@_atexit`, but This seems to be a valid concern.